os/drivers/lcd/lcd_dev.c

```diff
@@ -444,8 +447,10 @@ int lcddev_register(struct lcd_dev_s *dev)
 		lcddbg("ERROR: Failed to allocate memory for LCD flush swap buffer\n");
 		return -ENOMEM;
 	}
-
-	int pid = kernel_thread("LCD Frame flusing", 204, 8192, lcd_flushing_thread, NULL);
+	itoa((int)lcd_info, lcd_info_addr, 16);
+	task_info[0] = lcd_info_addr;
```

Comment:
The `lcd_info_addr` is a local pointer variable. Is it ok in lifecycle? When the flushing thread gets the arg, is it still valid?

Reply:
@sunghan-chang - Yes, it can happen that thread executes after scope of lcd_info_addr is finished. 
But to encounter this, if we use dynamic(malloc) lcd_info_addr, in thread function, the passed argument's address changes so we can't free it.
So should we keep static struct *lcd_info as global only ?

Reply:
I checked the function - thread_create.
It looks it's ok with local variable in lifecycle.

```
	/* Copy the argv[] array and all strings are to the task's stack.  Data on
	 * the stack is guaranteed to be accessible by the ask no matter what
	 * privilege mode the task runs in.
	 */

	return task_stackargsetup(tcb, argv);
```